src/stores/user-store.js

```diff
@@ -43,14 +44,27 @@ class UserStore extends EventEmitter {
         }
     }
 
+    _findUser (user) {
+        this._foundUsers = this._users.filter((anyUser) =>
```

Comment:
we didn't init `_foundUsers` in the constructor and we can even don't store them and just return from this method